### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by Sid-Uniyal**
Student Name: Siddharth Uniyal
Student Number: 45863608

This project will implement Pix2Pix GAN network.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be a lot more specific
- Should commit more frequently
- Code valid for Hard Difficulty

Other Feedback
- Good work on pix2pix
- Please follow python file  and folder naming convention e.g. `lorem_ipsum_123_abc.py`
- (OPTIONAL) Would put main logic in `if __name__ == '__main__'` for the sake of following python convention.

**Comment by rexsw** [comment]
Code looks good for hard difficulty

For next time;

Tensorflow has functions for image resizing and more so there’s no need to also use PIL and skimage in your project. 

Comments such as 
    discriminator = make_discriminator() # making a discriminator constructor
Add nothing to the readability of the code and should be avoided. Overall comments were well done

File: recognition/Pix2Pix/Driver.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```
Comment (shakes76) [inline recognition/Pix2Pix/Driver.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**